internal/environments/metadata.go

```diff
@@ -0,0 +1,13 @@
+package environments
+
+type Metadata struct {
+	Cluster     string `mapstructure:"ha-cluster,omitempty"`
+	Environment string `mapstructure:"sap-environment,omitempty"`
+	Landscape   string `mapstructure:"sap-landscape,omitempty"`
+	SAPSystem   string `mapstructure:"sap-system,omitempty"`
+}
```

Comment:
Isn't this node metadata, rather than environment?

Reply:
Ok, yes, it makes sense. I will move it to the `hosts` package (if there is not any strange import issue at least)